src/desktop/components/auth_modal/helpers.coffee

```diff
@@ -0,0 +1,43 @@
+sd = require('sharify').data
+gdprDisabled = sd.GDPR_COMPLIANCE_TEST is 'control'
```

Comment:
At what point does this line get evaluated? Transpile time, server runtime, or client runtime?

Reply:
Good question @yuki24. Based on [this](https://github.com/xtina-starr/force/blob/ae690e2e437e4819dc7423ce208440491c44608a/src/desktop/components/auth_modal/view.coffee#L24) I'd say client-side runtime.